src/components/drag-handle/drag-handle.tsx

```diff
@@ -0,0 +1,112 @@
+ * @exampleComponent limel-example-drag-handle-horizontal
+ *
+ * @private
+ */
+@Component({
+    tag: 'limel-drag-handle',
+    shadow: false,
+    styleUrl: 'drag-handle.scss',
+})
+export class DragHandleComponent {
+    /**
+     * The direction in which the drag handle can be used to reorder items.
+     * This affects both how the visual indicator is visualized,
+     * and how what helper label is shown in the tooltip.
+     */
+    @Prop({ reflect: true })
+    public dragDirection: 'vertical' | 'horizontal' = 'vertical';
+
+    /**
+     * The preferred direction for the tooltip to open.
+     * Defaults to 'left', as our recommended placement for a drag handle
+     * in the UI is on the far right side of draggable elements.
+     */
+    @Prop({ reflect: true })
+    public tooltipOpenDirection: OpenDirection = 'left';
+
+    /**
+     * Language to use for translations.
+     */
+    @Prop({ reflect: true })
+    public language: Languages = 'en';
+
+    private readonly dragHandleId = 'drag-handle-' + crypto.randomUUID();
+
+    public render() {
+        const tooltipLabel = translate.get(
+            'drag-handle.drag-to-reorder',
+            this.language
+        );
+        const ariaLabel = translate.get(
+            'drag-handle.drag-handle',
+            this.language
+        );
+
+        return (
+            <Host>
+                <button
+                    data-drag-handle
+                    type="button"
+                    class="limel-drag-handle"
+                    tabindex={0}
+                    aria-label={ariaLabel}
+                    id={this.dragHandleId}
+                    onClick={this.handleClick}
+                >
+                    <div
+                        class="drag-icon"
+                        role="presentation"
+                        aria-hidden="true"
+                    >
+                        <div key="1" />
+                        <div key="2" />
+                        <div key="3" />
+                        <div key="4" />
+                        <div key="5" />
+                        <div key="6" />
+                    </div>
+                    <limel-tooltip
+                        openDirection={this.tooltipOpenDirection}
+                        elementId={this.dragHandleId}
+                        label={tooltipLabel}
+                    />
+                </button>
+            </Host>
+        );
+    }
+
+    private readonly handleClick = (event: MouseEvent) => {
+        // This prevents the click (from keyboard space/enter or pointer) from
+        // reaching the parent item. The form template listens for clicks to
+        // toggle collapsible sections and invoke move/reorder actions.
+        // Without this, the synthetic click triggered by the keyboard would bubble,
+        // causing duplicate behavior alongside the custom keyboard reordering event.
+        event.preventDefault();
+        event.stopPropagation();
+    };
```

Comment:
If we have no keyboard support, it would be better to update this description, to avoid confusion.

In fact, if there is no keyboard-support, I would suggest setting `tabindex="-1"` on the drag-handle component, so it's not possible to tab to.